app/views/libraries/_library.json.jbuilder

```diff
@@ -0,0 +1,26 @@
+json.type 'libraries'
+json.id library.slug
+
+json.attributes do
+  json.extract! library, :name
+end
+
+json.relationships do
+  json.locations do
+
+    json.data do
+      json.array!(library.locations.with_hours) do |location|
+        json.type 'locations'
+        json.id "#{library.slug}/#{location.slug}"
+      end
+    end
+
+    json.links do
+      json.related library_locations_url(library)
```

Comment:
I can't seem to find any guidance from JSONAPI, but I assume that its understood that any related links will need to be requested with a JSON format or request header?

Reply:
I assume so too:

> JSON:API requires use of the JSON:API media type for exchanging data.